src/main/java/seedu/address/model/session/Interval.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.model.session;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Session's interval in FitEgo.
+ * Guarantees: immutable; is valid as declared in {@link #isValidInterval(int)}
+ */
+public class Interval {
+    public static final String MESSAGE_CONSTRAINTS = "Intervals can start at any time, and it have a positive duration";
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("E, dd MMM yyyy, HH:mm");
+
+    public final LocalDateTime start;
```

Comment:
Should this be non-public?
![Screen Shot 2020-09-29 at 5 58 08 PM](https://user-images.githubusercontent.com/55008082/94544026-57139200-027d-11eb-89c5-58dcfedd2e12.png)

Reply:
My DateTime formatter also different from yours
DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm")

Reply:
Oh right, I'll make it private